src/reducers/rubrics.js

```diff
@@ -1,21 +1,9 @@
 // TODO: Update to include array of rubrics
 
-const rubricsReducer = (state = {}, action) => {
+const rubricsReducer = (state = [], action) => {
```

Comment:
is rubricsReducer supposed to be an array?

Reply:
Each reducer is responsible for a piece of state.  The rubrics reducer produces the `rubrics` key in the state.  Since it is plural, it needs to be an array.